src/index.html

```diff
@@ -5,10 +5,44 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo-link">
+        <img class="logo-img" src="../src/images/logo.png" alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__list-items">
+            <a href="#" class="nav__list-link is-active">apple</a>
+          </li>
+          <li class="nav__list-items">
+            <a href="#" class="nav__list-link">samsung</a>
+          </li>
+          <li class="nav__list-items">
+            <a href="#" class="nav__list-link">смартфоны и телефоны</a>
```

Comment:
Add names of anchor links
```suggestion
            <a href="#apple" class="nav__list-link is-active">apple</a>
          </li>
          <li class="nav__list-items">
            <a href="#samsung" class="nav__list-link">samsung</a>
          </li>
          <li class="nav__list-items">
            <a href="#phones" class="nav__list-link">смартфоны и телефоны</a>
```